src/main/kotlin/no/nav/familie/ef/iverksett/felles/FamilieIntegrasjonerClient.kt

```diff
@@ -40,8 +43,14 @@ class FamilieIntegrasjonerClient(
         return response.getDataOrThrow()["aktørId"].toString()
     }
 
-    companion object {
+    fun hentNavEnhet(personident: String): Enhet? {
```

Comment:
Ettersom "default" navEnhet normalt er i ENF-kontekst - kunne denne funksjonen enten hete `hentNavEnhetForOppfølging` eller tatt inn `hentNavEnhetForTema(personIdent, "OPP")` for å tydeliggjøre skille mellom det vi i EF vanligvis tenker for å hente enhet?

Reply:
Godt poeng. Renamet til `hentNavEnhetForOppfølging`